### ISSUE TITLE
Screen readers behaving differently

### ISSUE THREAD

**Issue by carolina-nakamura** [flutter/flutter#157661]
Hello! In my application I have a line with a text content and trailing buttons, like the image:

<img width="376" alt="Screenshot 2024-10-25 at 23 27 35" src="https://github.com/user-attachments/assets/c90f42c5-9c00-4121-bf32-643588dd9653">

When the + or - buttons are clicked, the text content refreshes.
I was doing tests with screen readers and TallBack and VoiceOver behave differently when after the buttons are clicked.

With VoiceOver the new values are read after the click, but Tall Back doesn't have any feedback.

**Comment by darshankawar** [comment]
@carolina-nakamura 
Can you provide a complete runnable reproducible code sample along with `flutter doctor -v` ?


> With VoiceOver the new values are read after the click, but Tall Back doesn't have any feedback.

This indicates the reported behavior is on Android ?
